model/distro/distro.go

```diff
@@ -42,6 +44,7 @@ type Distro struct {
 	DisableShallowClone   bool                    `bson:"disable_shallow_clone" json:"disable_shallow_clone" mapstructure:"disable_shallow_clone"`
 	UseLegacyAgent        bool                    `bson:"use_legacy_agent" json:"use_legacy_agent" mapstructure:"use_legacy_agent"`
 	Note                  string                  `bson:"note" json:"note" mapstructure:"note"`
+	MountScript           string                  `bson:"mount_script" json:"mount_script" mapstructure:"mount_script"`
```

Comment:
can this be MountCommand because script implies shell or something, and we really only want this to be a single jasper op without a shell dependency.

Reply:
done.

Reply:
did you miss a push?

Reply:
this no longer applies (the variable was removed)